models/issue_comment.go

```diff
@@ -514,17 +504,17 @@ type CreateCommentOptions struct {
 	Issue *Issue
 	Label *Label
 
-	OldMilestoneID int64
-	MilestoneID    int64
-	OldAssigneeID  int64
-	AssigneeID     int64
-	OldTitle       string
-	NewTitle       string
-	CommitID       int64
-	CommitSHA      string
-	LineNum        int64
-	Content        string
-	Attachments    []string // UUIDs of attachments
+	OldMilestoneID  int64
+	MilestoneID     int64
+	AssigneeID      int64
```

Comment:
Maybe `AssigneeID` -> `AssigneeIDs` is better? So when I assign multiple users, it will create only one comment.

Reply:
This would mean changing the way I currently assign users as I currently loop through all users passed and then add each one seperately + creating the comments. It probably would also require redoing some of the frontend-UI.